{"name": "Hoover does not move if driving into a wall -- @1.4 ", "status": "failed", "statusDetails": {"message": "AssertionError: Expected 0 but got 1\n", "trace": "  File \"/home/luis/.local/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/luis/.local/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/hoover_functional_steps.py\", line 42, in step_impl_then_hoover_position\n    assert context.response_body[\"patches\"] == int(cleaned_patches), f\"Expected {int(cleaned_patches)} but got {context.response_body['patches']}\"\n"}, "steps": [{"name": "Given a room with dimensions \"5\" by \"5\"", "status": "passed", "start": 1682670788096, "stop": 1682670788096}, {"name": "And a hoover starting at position \"4\", \"4\"", "status": "passed", "start": 1682670788096, "stop": 1682670788096}, {"name": "And the following patches of dirt: \"1,0;2,2;2,3\"", "status": "passed", "start": 1682670788096, "stop": 1682670788096}, {"name": "When the hoover moves according to \"E\"", "status": "passed", "start": 1682670788096, "stop": 1682670788102}, {"name": "Then the final position of the hoover should be \"4\", \"4\"", "status": "passed", "start": 1682670788102, "stop": 1682670788102}, {"name": "And the number of cleaned patches should be \"0\"", "status": "failed", "statusDetails": {"message": "AssertionError: Expected 0 but got 1\n", "trace": "  File \"/home/luis/.local/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/luis/.local/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/hoover_functional_steps.py\", line 42, in step_impl_then_hoover_position\n    assert context.response_body[\"patches\"] == int(cleaned_patches), f\"Expected {int(cleaned_patches)} but got {context.response_body['patches']}\"\n"}, "start": 1682670788102, "stop": 1682670788103}], "parameters": [{"name": "room_x", "value": "5"}, {"name": "room_y", "value": "5"}, {"name": "start_x", "value": "4"}, {"name": "start_y", "value": "4"}, {"name": "dirt_positions", "value": "1,0;2,2;2,3"}, {"name": "instructions", "value": "E"}, {"name": "final_x", "value": "4"}, {"name": "final_y", "value": "4"}, {"name": "cleaned_patches", "value": "0"}], "start": 1682670788094, "stop": 1682670788103, "uuid": "6717b415-fdf8-4b0c-94c2-0f75e96ff4df", "historyId": "336c0cbe39d748fd3132ea993e1ef0af", "fullName": "Hoover service testing: Hoover does not move if driving into a wall", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "Hoover service testing"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}